packages/tocco-ui/src/CodeEditor/AceEditor.js

```diff
@@ -72,26 +72,26 @@ const AceEditor = props => {
     showGutter = true,
     editorOptions = {}
   } = props
-  const [reference, setReference] = useState(null)
-  const [editor, setEditor] = useState(null)
+  const containerReference = useRef(null)
+  const editorReference = useRef(null)
 
   useEffect(() => {
-    if (reference) {
-      const aceEditor = ace.edit(reference)
+    if (containerReference.current) {
+      const aceEditor = ace.edit(containerReference.current)
       aceEditor.getSession().setValue(value)
       aceEditor.on('change', () => onChange(aceEditor.getValue()))
       setEditorConfiguration(aceEditor, props)
-      setEditor(aceEditor)
+      editorReference.current = aceEditor
 
       return () => aceEditor.destroy()
     }
-  }, [reference])
+  }, [containerReference])
```

Comment:
This not needed anymore => you can use an empty array `[]`
it will then run this once "on mount"